src/grelu/io/motifs.py

```diff
@@ -0,0 +1,158 @@
+"""
+Functions related to reading and writing MEME files
+"""
+
+from typing import Dict, List, Optional
+
+import numpy as np
+
+
+def read_meme_file(
```

Comment:
logic seems to overlap with https://github.com/jmschrei/tfmodisco-lite/blob/3c6e38f3ad5df80c55bd4e8c7c2a531ee0a2b316/modiscolite/report.py#L22, maybe better to call that to avoid redundancy?

Reply:
alternately https://github.com/jmschrei/tangermeme/blob/691d77a6df9ac474dfa9ec2fa552fb0ee4280199/tangermeme/io.py#L392 might be better

Reply:
done